converters/fromZigbee.js

```diff
@@ -4086,7 +4086,12 @@ const converters = {
             case tuya.dataPoints.moesSvalvePosition:
                 return {position: value};
             case tuya.dataPoints.moesScompensationTempSet:
-                return {local_temperature_calibration: value};
+                return {
+                    local_temperature_calibration: value,
+                    ...(meta && meta.state && meta.state.local_temperature != null && meta.state.local_temperature_calibration != null) ?
+                        {local_temperature: meta.state.local_temperature + (value - meta.state.local_temperature_calibration)} :
```

Comment:
Instead of `meta.state.local_temperature_calibration`, why not use `value`?

Reply:
If you are referring to the fourth line, it's because I need to subtract the old calibration value before adding the new one (assuming we know the old calibration value, hence the previous null checks). Do you want me to add a comment? I realise the code can be obscure, maybe a justification re the valve not updating the temperature is in order here.

Reply:
Adding a note would be good. 

So if I understand correctly, currently the `local_temperature` is correct but it takes some time to update after changing the `local_temperature_calibration`?

Reply:
> So if I understand correctly, currently the `local_temperature` is correct but it takes some time to update after changing the `local_temperature_calibration`?

Yes, the valve applies the new calibration when it comes to opening or closing the valve, and it shows the re-calibrated value on its display instantly. It just doesn't report the new value over Zigbee until the (uncalibrated) measurement itself change.